frontend/__tests__/units.js

```diff
@@ -291,11 +291,11 @@ describe('validate', () => {
   });
 
   it('cpu', () => {
-    ['32', '32m', '32K'].forEach(v => {
+    ['32', '32m'].forEach(v => {
       expect(validate.CPU(v)).toEqual(undefined);
```

Comment:
little confused on this, isn't '32m' a valid CPU quantity?  why does it equal `undefined`?